unit-tests/live/streaming/test-jpeg-compressed-format.py

```diff
@@ -0,0 +1,40 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2025 Intel Corporation. All Rights Reserved.
+
+# test:device D555
+
+
+import pyrealsense2 as rs
+from rspy import test
+from rspy import log
+
+
+with test.closure( "Check jpeg format support"):
+    ctx = rs.context({"format-conversion":"raw"})
```

Comment:
What does this means?
Not familiar with this syntax

Reply:
Passing a json object with setting to the context. Here asking for no format conversions to verify camera streams JPEG.